crates/sui-core/src/quorum_driver/mod.rs

```diff
@@ -219,7 +243,7 @@ where
             .await;
 
         match result {
-            Err(SuiError::QuorumFailedToProcessTransaction {
+            Err(QuorumExecuteTransactionError {
```

Comment:
I think if it would be more accurate by naming the error `QuorumSignTransactionError` or something similar.